hazelcast-client/src/test/java/com/hazelcast/client/impl/ClientTestUtil.java

```diff
@@ -0,0 +1,19 @@
+package com.hazelcast.client.impl;
+
+import com.hazelcast.core.HazelcastInstance;
+import org.junit.Ignore;
+
+@Ignore("not a JUnit test")
```

Comment:
I assume the classname matches a convention for tests -> Surefire wanted to execute it -> it fails because JUnit compained "No Test Found".